go/launcher/webdriver/webdriver_error.go

```diff
@@ -146,14 +146,14 @@ func errDatum(resp *jsonResp) errorDatum {
 			}
 		}
 	}
-	if resp.Status != 0 {
+	if resp.Status != nil && *resp.Status != 0 {
 		for _, cand := range errorData {
-			if cand.Status == resp.Status {
+			if cand.Status == *resp.Status {
 				return cand
 			}
 		}
 	}
-	return errorDatum{resp.Status, resp.Error, 500, false}
+	return errorDatum{*resp.Status, resp.Error, 500, false}
```

Comment:
I set the error status to -1 (similar to other error types that don't match) if status is unknown. This gets turned into a 13 (unknown error) when serialized.